.editorconfig

```diff
@@ -0,0 +1,11 @@
+# top-most EditorConfig file
```

Comment:
This file should probably be .gitignore'd

Reply:
I disagree, keeping this file means the coding style is maintained between different editors and IDEs.